lambda-template.yaml

```diff
@@ -209,9 +209,7 @@ Resources:
     Metadata:
       BuildMethod: go1.x
     Properties:
-      CodeUri:
-        Bucket: !FindInMap [ RegionToS3Bucket, !Ref 'AWS::Region', BucketArn ]
-        Key: 'new-relic-log-forwarder-folder/new-relic-log-forwarder.zip'
+      CodeUri: src/
```

Comment:
Also need to change all templates which refer to `lambda-template.yaml` as SAM package would try to publish lambda code again. Reference to `lambda-template.yaml` needs to be updated to S3 object uri.

Reply:
As there are many changes involved in supporting this I am debating whether we should make this change or just update it manually whenever we have to test changes in lambda code.

Reply:
@maya-jha regarding point. 1 -> I didn't understand what changes are needed as in all the parent templates lambda-template is referenced as below. So what needs to be changed here? I see that this will work as expected locally when we use sam publish as well. 
'''
  NewRelicLogsLoggingLambda:
    Type: 'AWS::CloudFormation::Stack'
    Properties:
      TemplateURL: lambda-template.yaml
'''

Reply:
Regarding point 2. We need this change at least in one place, in ci/cd or in some documentation like README because this will be a blocker for any new developer getting onboarded.